gallery/src/components/NxTag/NxTagNarrowExample.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useState } from 'react';
+import { NxTag, NxSelectableTag, NX_TAG_COLORS_TYPE } from '@sonatype/react-shared-components';
+
+interface TagInfo {
+  color?: NX_TAG_COLORS_TYPE;
+  value: string;
+}
+
+const tags: TagInfo[] = [
+  { color: undefined, value: 'Default' },
+  { color: 'purple', value: 'Purple' },
+  {
+    color: 'pink',
+    value: `Pink - artisinal singularity tiger-team BASE jump meta-soul-delay network footage garage spook towards
+        tiger-team weathered chrome warehouse`
+  },
+  { color: 'blue', value: 'Blue' },
+  { color: 'red', value: 'Red' },
+  { color: 'green', value: 'Green' },
+  { color: 'orange', value: 'Orange' },
+  { color: 'yellow', value: 'Yellow' },
+  { color: 'lime', value: 'Lime' },
+  { color: 'indigo', value: 'Indigo' }
+];
+
+function NxTagNarrowExample() {
+  const [selectedTags, setSelectedTags] = useState<Set<TagInfo>>(new Set());
+
+  function toggleTag(tag: TagInfo) {
+    const newSet = new Set(selectedTags);
+
+    if (newSet.has(tag)) {
+      newSet.delete(tag);
+    }
+    else {
+      newSet.add(tag);
+    }
+
+    setSelectedTags(newSet);
+  }
+
+  return (
+    <div style={{width: '250px'}}>
```

Comment:
I'd give this a simple border to make it clearer what's happening.

Reply:
> I'd give this a simple border to make it clearer what's happening.

https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/3cd25115348de47a8a152a4f80eb249681e1e9c9